src/librustdoc/html/render/mod.rs

```diff
@@ -1844,7 +1844,7 @@ fn document(w: &mut Buffer, cx: &Context, item: &clean::Item, parent: Option<&cl
     if let Some(ref name) = item.name {
         info!("Documenting {}", name);
     }
-    document_stability(w, cx, item, false, parent);
+    document_item_info(w, cx, item, false, parent);
     document_full(w, item, cx, "", false);
```

Comment:
Why are these separate? Are there cases when you want one but not the other?

Reply:
No, there isn't. Let's call `document_item_info` inside document full. Great catch!

Reply:
Hum actually we can't really do that in a straightforward way because in some cases, we add the information on the parent. For example:

```rust
document_item_info(w, cx, item, is_hidden, Some(parent));
if show_def_docs {
    document_full(w, item, cx, "", is_hidden);
}
```

And it's in multiple places. `document_short` has the same "problem".

Reply:
Could you pass in `show_def_docs` as a parameter? Then it would have the same behavior, but make it hard to forget about `document_item_info`.

Reply:
Fine by me!